41_Redux/parettool/src/redux/productsSlice.js

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const productsSlice = createSlice({
+  name: 'products',
+  initialState: {
+    products: [],
+    product: [],
+  },
+
+  reducers: {
+    addNewProduct: (state, action) => {
+      state.products = [...state.products, action.payload]
+    },
+   
+    addProductToCart: (state, action) => {
+      state.product = [...state.product, action.payload]
+    },
+
+    setProductToCart: (state, action) => {

```

Comment:
I think the name of this function is wrong?
why Cart?
should be just `setProducts`